lib/chromedriver.js

```diff
@@ -134,24 +135,39 @@ class Chromedriver extends events.EventEmitter {
   async getChromedrivers (mapping) {
     // go through the versions available
     const executables = await fs.glob(`${this.executableDir}/*`);
+    log.debug(`Found ${executables.length} executable${executables.length === 1 ? '': 's'} ` +
+      `in '${this.executableDir}'`);
     const cds = (await asyncmap(executables, async function (executable) {
+      let stdout;
+      let errMsg = `Cannot retrieve version string from '${path.basename(executable)}' Chromedriver binary. ` +
```

Comment:
Having the error message pre-made feels weird to me. I think it would read better to have a function that would return the correct string. Something like
```js
const errMsg = function errMsg (executable, err = {}) {
  return `Cannot retrieve version string from '${path.basename(executable)}' Chromedriver binary. ` +
         `Make sure it returns a valid version string in response to '--version' command line argument. ` +
         `Original error: ${err.message | ''}; ` +
         `stdout: ${err.stdout || ''}; ` +
         `stderr: ${err.stderr || ''}.`;
};
```

Reply:
changed